src/main/scala/com/fulcrumgenomics/util/NcbiRefSeqGffSource.scala

```diff
@@ -226,27 +226,30 @@ class NcbiRefSeqGffSource private(lines: Iterator[String],
     * */
   private def parseGene(chrom: String, geneRec: GffRecord, iter: BetterBufferedIterator[GffRecord]): Option[Gene] = {
     val txsBuilder = Seq.newBuilder[Transcript]
+    val biotype = GeneBiotype(geneRec("gene_biotype"))
 
     // Loop while the next record is a child of this gene and is "transcript-like", which includes any
     // direct children that define the attribute "transcript_id", not just records with kind=transcript.
-    while (iter.hasNext && iter.head.parentId.contains(geneRec.id) && iter.head.has("transcript_id")) {
+    while (iter.hasNext && iter.head.parentId.contains(geneRec.id)) {
```

Comment:
1. Why not use `headOption`?
```suggestion
    while (iter.headOption.exists(_.contains(geneRec.id))) {
```
2. I think the comment about "transcript-like" is out of date

Reply:
Good catch on the comment.  I tend to prefer `iter.hasNext` perhaps just due to familiarity.

Reply:
I always though that scala folks wanted to avoid `.head`, `.get` and so on, preferring to rely on options